src/excmds.ts

```diff
@@ -2021,6 +2021,26 @@ export function fillcmdline_notrail(...strarr: string[]) {
     messageActiveTab("commandline_frame", "fillcmdline", [str, trailspace])
 }
 
+/** Show and fill the command line without focusing it */
+//#background
+export function fillcmdline_nofocus(...strarr: string[]) {
+    showcmdline(false)
+    return messageActiveTab("commandline_frame", "fillcmdline", [strarr.join(" "), false, false])
+}
+
+/** Shows str in the command line for ms milliseconds */
+//#background
+export async function fillcmdline_tmp(ms: string, ...strarr: string[]) {
+    let milliseconds = parseInt(ms)
+    await fillcmdline_nofocus(...strarr)
+    return new Promise(resolve =>
+        setTimeout(async () => {
+            await messageActiveTab("commandline_frame", "hide_and_clear", [])
```

Comment:
Potential issue here - does this timer get cancelled if I start entering things into the command line?

Reply:
I think I disagree. Messages will need to be displayed for about 5 seconds to be readable, this means that if I e.g. press `t` in that time, the command line will close seemingly at random.